tutorials/27_First_RAG_Pipeline.ipynb

```diff
@@ -1,1446 +1,1316 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    print(response["llm"]["replies"][0])
</pre><p>we can change this part as <code>print(response["llm"]["replies"][0].text)</code> -&gt; I believe the ChatMessage <code>content</code> will be <code>text</code>     </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/366/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>